### PR TITLE
513 Add content to routepath list objects

### LINKED ISSUE #513: RPV - 6 Content to RP tab 2 blocks
Plan view for nodes and links under the routepath view list. Content should contain most important info, and allow modifying info that concerns routepath. Also should have link to the solmu and link view with full information in network mode.

### PR DESCRIPTION
**Pull request by kronken**
Closes #513

### CODE REVIEW

File: src/components/sidebar/routePathView/routePathListTab/RoutePathListNode.tsx [hunk 1]
```diff
@@ -17,66 +23,134 @@ interface IRoutePathListNodeState {
...
+                        s.nodeIcon,
+                        routePathLink.isStartNodeTimeAlignmentStop
+                            ? s.timeAlignmentIcon
+                            : undefined)
+                    }
+            />
+        );
+
+        if (node.type === NodeType.MUNICIPALITY_BORDER) {
+            icon = this.addBorder(icon, '#c900ff');
```
Comment (kronken) [inline src/components/sidebar/routePathView/routePathListTab/RoutePathListNode.tsx]: import there?

File: src/components/sidebar/routePathView/routePathListTab/RoutePathListLink.tsx [hunk 2]
```diff
@@ -1,21 +1,74 @@
...
     extends React.Component<IRoutePathListLinkProps> {
+    private renderNodeHeaderIcon = () => <div className={s.linkIcon} />;
+
+    private renderRoutePathLinkView = (rpLink: IRoutePathLink) => {
+        return (
+            <div className={s.nodeContent}>
+                Reitinlinkin tiedot
+                <div className={s.flexRow}>
+                    <InputContainer
+                        label='JÄRJESTYS NUMERO'
```
Comment (kronken) [inline src/components/sidebar/routePathView/routePathListTab/RoutePathListLink.tsx]: JÄRJESTYSNUMERO?

File: src/components/sidebar/routePathView/routePathListTab/RoutePathListLink.tsx [hunk 3]
```diff
@@ -1,22 +1,74 @@
...
 
 interface IRoutePathListLinkProps {
+    routePathStore?: RoutePathStore;
+    previousNode?: INode;
     routePathLink: IRoutePathLink;
+    nextNode?: INode;
 }
 
+@observer
+@inject('routePathStore')
```
Comment (neodyymi) [inline src/components/sidebar/routePathView/routePathListTab/RoutePathListLink.tsx]: Switch order of these decorators.
`observer` should be applied first and now the decorators would translate to
`observer(inject('routePathStore)(RoutePathListLink))`

Comment (neodyymi) [reply src/components/sidebar/routePathView/routePathListTab/RoutePathListLink.tsx]: This issue also appears in _RouteLayer.tsx_

**Pull request merged**